libexec/askiy.js

```diff
@@ -1,22 +1,30 @@
+const fs = require('fs')
+const fsPromise = fs.promises
+
+// todo: use Pathlike from fs once we have typescript
```

Comment:
that is, if we are doing it at all...On my talk with Antti he wants to have click-in-text work again so that he can test it out. I'll do that first and later see if I have time for typescript setup